task3/APP/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+
+  #-----------------web app-----------------
+  webapp:
```

Comment:
no need to build the app again, you should pull it from the docker-registry from the build in the last task